testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -59,6 +60,7 @@ use crate::{neon_node, ChainTip};
 const ABORT_TRY_AGAIN_MS: u64 = 200;
 /// If the signers have not responded to a block proposal, how long should
 ///  the miner thread sleep before trying again?
+#[allow(unused)]
 const WAIT_FOR_SIGNERS_MS: u64 = 200;
```

Comment:
Should we just delete this then?

Reply:
Yep, done in [c50a930](https://github.com/stacks-network/stacks-core/pull/4481/commits/c50a93088d7c0261f1dbe31ab24b95028a038447)